discord/types/guild.py

```diff
@@ -120,6 +121,7 @@ class Guild(_BaseGuildPreview, _GuildOptional):
     roles: List[Role]
     mfa_level: MFALevel
     nsfw: bool
```

Comment:
```suggestion
```

This apparently got removed. The guild property will need to get removed as well.